core/src/main/java/com/linecorp/armeria/client/DnsResolverGroupBuilder.java

```diff
@@ -292,6 +295,16 @@ public DnsResolverGroupBuilder decodeIdn(boolean decodeIdn) {
         return this;
     }
 
+    /**
+     * Sets MeterRegistry.
+     * @param metricRegistry {@link PrometheusMeterRegistry}.
+     * @return DnsResolverGroupBuilder.
+     */
+    public DnsResolverGroupBuilder metricRegistry(PrometheusMeterRegistry metricRegistry) {
```

Comment:
- Should accept `MeterRegistry`.
- Could be package-private?

Reply:
Done